gui/resources.xml

```diff
@@ -210,15 +211,15 @@ Vocabulary
           <!-- Code in document.py takes care of ltr/rtl details -->
           <signal name="activate" handler="undo_cb"/>
         </object>
-        <accelerator key="z"/>
+        <accelerator key="z" modifiers="GDK_CONTROL_MASK"/>
       </child>
       <child>
         <object class="GtkAction" id="Redo">
           <property name="icon_name">mypaint-redo-ltr-symbolic</property>
           <!-- Code in document.py takes care of ltr/rtl details -->
           <signal name="activate" handler="redo_cb"/>
         </object>
-        <accelerator key="y"/>
+        <accelerator key="z" modifiers="GDK_CONTROL_MASK | GDK_SHIFT_MASK"/>
```

Comment:
Hm, Ctrl-Y for redo is quite common I think? What I wanted at the time was to reach all common features with a single finger, that's why I removed the Ctrl. Two-key combos are a bit more difficult to do single-handed.